cibuildwheel/__main__.py

```diff
@@ -198,6 +198,9 @@ def detect_obsolete_options():
         if deprecated in os.environ and 'manylinux1' in os.environ[deprecated]:
             print("Build identifiers with 'manylinux1' been deprecated. Replacing all occurences of 'manylinux1' by 'manylinux' in the option '{}'".format(deprecated))
```

Comment:
`"Build identifiers with 'manylinux1' have been deprecated."` Not sure how we've overlooked that typo before.